CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Immobilienscout Changelog
+
+## v0.0.3 [Unreleased]
+
```

Comment:
So this upcoming 0.3.3 release already includes a few changes that were not released yet right? The should be in here already. In the future, the changelog should be updated directly in the PR where changes are made.

Reply:
Yes, this version will have the new JSON structure response for the ScoutReport from Immobilienscout. I will add it here then.